package/base-files/files/sbin/wifi

```diff
@@ -194,8 +193,8 @@ set_wifi_down() {
 	local cfg="$1"
 	local vifs vif vifstr
 
-	[ -f "/var/run/wifi-${cfg}.pid" ] &&
-		kill "$(cat "/var/run/wifi-${cfg}.pid")" 2>/dev/null
+	[ -f "/var/run/wifi-${cfg}.pid" ] \
+		&& kill "$(cat "/var/run/wifi-${cfg}.pid")" 2> /dev/null
```

Comment:
useless cat here. kill can read from stdin.

Reply:
Shellcheck tells me it can not.

Reply:
It doesn’t warn on it but kill definitely accepts stdin.

Reply:
I think shellcheck warns only on shell builtins.

Reply:
```
kill < pid
ash: you need to specify whom to kill
```
Are you sure?

Reply:
Huh. Just tested it. It seems kill doesn't actually work with stdin. Wonder how I got it working...